src/routes/TraderForm/Form/MarginAmountField/index.tsx

```diff
@@ -19,6 +20,9 @@ export const MarginAmountField: React.FunctionComponent<NotionalAmountProps> = (
   const infoPostSwap = useAppSelector(selectInfoPostSwap);
   const aMM = useAppSelector(selectSwapFormAMM);
 
+  const marginRequiredError = marginAmount.error !== null && marginAmount.error !== 'WLT';
```

Comment:
perhaps move this to selector as well. That returns string or undefined then no need for the cast `as string` below.

Reply:
This is a boolean, we would still need casting

Reply:
[753f5ee](https://github.com/Voltz-Protocol/voltz-ui/pull/576/commits/753f5ee4ee927fec2af52222bba233742d903d88)